src/components/Header/Header.js

```diff
@@ -137,7 +138,8 @@ export default function Header() {
       return menuHeaders
         .filter(({ href }) => href !== '/logout' && href !== '/account')
         .map(({ id, href, label }) => {
-          let color = label === 'Join Us!' ? '#50E3C2' : 'white';
+          // Change "Join Us!" to another color
```

Comment:
remove the comment

Reply:
done! d2659116d3b701c0fb650123af1253b4667c77fe